src/pages/configurePatch/configurePatchCore/ConfigureTasks.tsx

```diff
@@ -7,6 +7,8 @@ import Checkbox from "@leafygreen-ui/checkbox";
 import { css } from "@emotion/core";
 import isEmpty from "lodash/isEmpty";
 import { Button } from "components/Button";
+import get from "lodash/get";
+import every from "lodash/every";
```

Comment:
I think you forgot to commit the package.json I dont see this package in the package.json. https://github.com/evergreen-ci/spruce/blob/master/package.json#L44